project/sites/hseni/themes/hseni_theme/src/scss/3_components/navigation/main-menu.scss

```diff
@@ -0,0 +1,12 @@
+/*
+ * @file
+ * Provides the layout styles for the main nav.
+ */
+
+.main-menu .nav-main {
+  background: #044a96;
```

Comment:
There should now be variables in the root.scss file to change these colours.

Reply:
Thanks Adam, i have deleted this file now since it is not required